core/src/com/unciv/logic/civilization/diplomacy/CityStateFunctions.kt

```diff
@@ -677,7 +677,7 @@ class CityStateFunctions(val civInfo: Civilization) {
     fun getCityStateResourcesForAlly() = ResourceSupplyList().apply {
         for (city in civInfo.cities) {
             // IGNORE the fact that they consume their own resources - #4769
-            addPositiveByResource(city.getCityResources(), Constants.cityStates)
+            addPositiveByResource(city.getResourcesGeneratedByCity(), Constants.cityStates)
```

Comment:
Why is this not Civ resources? City states can capture unrazeable cities. Mods may give a city state a resource